packages/core/src/schemas/releaseJsonMetadata.json

```diff
@@ -116,6 +116,9 @@
                   },
                   "sha256": {
                     "type": "string"
+                  },
+                  "uri": {
```

Comment:
What fix allowed us to bring this back? I recall it was removed because it was causing crashes, but no detail on what the failure was.

Reply:
Using the `MetaplexFileReplacer` :)